pkg/registry/authorization/localsubjectaccessreview/rest.go

```diff
@@ -57,10 +57,10 @@ func (r *REST) Create(ctx genericapirequest.Context, obj runtime.Object, include
 	}
 
 	authorizationAttributes := authorizationutil.AuthorizationAttributesFrom(localSubjectAccessReview.Spec)
-	allowed, reason, evaluationErr := r.authorizer.Authorize(authorizationAttributes)
+	decision, reason, evaluationErr := r.authorizer.Authorize(authorizationAttributes)
 
 	localSubjectAccessReview.Status = authorizationapi.SubjectAccessReviewStatus{
-		Allowed: allowed,
+		Allowed: (decision == authorizer.DecisionAllow),
```

Comment:
subject access review must stay as expressive as our authorizer interface. A second field might be the cleanest way to accomplish this:
```
Allowed: (decision == authorizer.DecisionAllow)
Denied: (decision == authorizer.DecisionDeny)
```

Reply:
Why not just directly add the decision field to the response?

Reply:
I'm worried about painting ourselves into a corner with another bool.

Reply:
possibly. by the "Happy families are all alike; every unhappy family is unhappy in its own way." theory of API design, we only really need to distinguish flavors of denials.

Reply:
I think that the flavor can be conveyed effectively in the reason so I'm going with a bool